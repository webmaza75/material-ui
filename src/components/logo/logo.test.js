import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Logo from './logo.jsx';

describe(`Logo`, () => {
  it(`Logo renders correctly`, () => {
    const renderer = new ShallowRenderer();
    const result = renderer
      .render(<Logo />);

    expect(result).toMatchSnapshot();
  });
});