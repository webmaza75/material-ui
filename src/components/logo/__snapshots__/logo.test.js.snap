// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Logo renders correctly 1`] = `
<div
  className="makeStyles-logo-1"
>
  LOGO
</div>
`;
